Makefile

```diff
@@ -25,6 +25,17 @@ cover:
 showcover:
 	go tool cover -html=coverage.txt
 
+acc-cover:
+	rm -fr ./acceptance/build/cover/
+	CLI_GOCOVERDIR=build/cover go test ./acceptance
+	rm -fr ./acceptance/build/cover-merged/
+	mkdir -p acceptance/build/cover-merged/
+	go tool covdata merge -i $$(printf '%s,' acceptance/build/cover/* | sed 's/,$$//') -o acceptance/build/cover-merged/
+	go tool covdata textfmt -i acceptance/build/cover-merged -o coverage-acceptance.txt
```

Comment:
This can be implemented in Go so we run `go test ./acceptance -coverprofile=coverage-acceptance.txt` (see gotestsum line above for reference). The path manipulation + expansion + sed here feels fragile.

Can be a possible (not required) follow-up.